src/web/core/src/main/resources/GeoServerApplication.properties

```diff
@@ -1006,6 +1006,10 @@ DimensionEditor.referenceValue.invalidNearestTimeReferenceValue = Reference time
 DimensionEditor.referenceValue.invalidTimeReferenceValue = Reference time value must be an ISO 8601 DateTime
 DimensionEditor.referenceValue.invalidElevationReferenceValue = Reference elevation value must be numeric
 DimensionEditor.referenceValue.emptyReferenceValue = Reference value cannot not be empty
+DimensionEditor.invalidAcceptableInterval = Invalid syntax for acceptable interval: '${actual}'. Must be empty, a single value, or two values separated by slash. Time values must use the ISO period sytanx (e.g., PT1H)
```

Comment:
Cool, I always forget about wicket parameterization support ...

Reply:
Yeah, I normally use positional variables but seems that for error messages one has to use named ones.